umh-core/pkg/diagnosis/engine_episode_test.go

```
@@ -0,0 +1,300 @@
+
+	// at names a tick by its index. The table interval is one second, so tick i
+	// is i seconds into the episode, and every instant asserted below is stated
+	// as the tick it belongs to.
+	at := func(i int) time.Time { return base.Add(time.Duration(i) * time.Second) }
+
+	// names lists one level of a fired set by signal name, so a phase can state
+	// the shape of that level in one line.
+	names := func(fired []Fired) []string {
+		out := make([]string, len(fired))
+		for i, f := range fired {
+			out[i] = f.Identity.Signal
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "Identity" from selector

(staticcheck)